changes/ocd-4520.sql

```diff
@@ -0,0 +1,39 @@
+insert into openchpl.report_metadata (environment, title, report_key, url, height, display_order, last_modified_user)
+select 'DEV', 'Non-conformity Counts', 'Non-conformityCounts',
```

Comment:
```suggestion
select 'DEV', 'Non-Conformity Counts', 'Non-conformityCounts',
```

I believe this is the correct capitalization